.github/workflows/tests-integration-reusable.yml

```
@@ -22,17 +22,29 @@ on:
       lease-id:
         required: true
         type: string
+      lease-namespace:
+        required: false
+        type: string
+        default: 'tas-devex'
       gitRef:
         type: string
         default: ${{github.event.workflow_run.head_sha}}
-        
+      nodes:
+        type: string
+        default: "8"
+
+env:
+  NODES: ${{ inputs.nodes }}
+  FLAKE_ATTEMPTS: ${{ vars.TEST_FLAKE_ATTEMPTS || '2' }}
```

There is no need to change the name of the variables here, it might create more confusion.
```suggestion
  TEST_FLAKE_ATTEMPTS: ${{ vars.TEST_FLAKE_ATTEMPTS || '2' }}
```

---

I'll keep it aligned with Ginkgo. What do you think about changing it from 2 to 4?